### DISCUSSION TITLE
`toggle_tab_overview` action doesn't seem to work if `gtk-titlebar = false`

### DISCUSSION THREAD

**Discussion by goproslowyo** [ghostty-org/ghostty#3910] [Feature Requests, Ideas] [answered]
Seemingly a bug:

`keybind = cmd+\=toggle_tab_overview` action doesn't seem to work if `gtk-titlebar = false`

I've bound `toggle_tab_overview` to `cmd+\` (I'm on Linux for this case) but the tab overview doesn't open when the titlebar is disabled.

---

Separately, I noticed that when we're in the tab_overview mode that keyboard input still seems to be sent to the active terminal surface (e.g. using arrows or typing text was showing up in the active terminal and in the tiny tab_overview preview thumbnail as well)

edit:

<details>
<summary>Ghostty version info</summary>
$ ghostty +version
Ghostty 1.0.1-main+6508fec9

Version
  - version: 1.0.1-main+6508fec9
  - channel: tip
Build Config
  - Zig version: 0.13.0
  - build mode : builtin.OptimizeMode.ReleaseFast
  - app runtime: apprt.Runtime.gtk
  - font engine: font.main.Backend.fontconfig_freetype
  - renderer   : renderer.OpenGL
  - libxev     : main.Backend.io_uring
  - GTK version:
    build      : 4.14.2
    runtime    : 4.14.2
  - libadwaita : enabled
    build      : 1.5.0
    runtime    : 1.5.0
  - libX11     : enabled
</details>

**Comment by goproslowyo** [comment, marked as answer]
I thiiink this was resolved by #3940  -- this works now. Thank you @tristan957

**Reply by byqzna** [reply]
The toggle_tab_overview works but now it doesn't show the system theme head bar anymore.  I'd rather have that back and forego the toggle_tab_overview. ( because of resizing and the minimize,maximize,close buttons)
Arch>Kde

**Reply by tristan957** [reply]
I don't know what that means.
